_distro_map.yml

```diff
@@ -187,6 +187,19 @@ openshift-rosa:
     rosa-preview:
```

Comment:
Would it be too much effort to change the dir to `rosa-classic`?

Reply:
docs.os folks please do correct me, but I'm pretty sure this defines the directory name to save the built content in, and uses that same dir name as part of the URL for docs.openshift.com, right?

So we should be able to define one of these for rosa-classic and one for rosa, and just ensure we're using the right build conditions for each distro, correct?

Reply:
It sounds like team consensus is that we're okay with keeping the ROSA with classic architecture dir  name as `rosa` and the new ROSA distro having the dir name `rosa-hcp`.